base/threadingconstructs.jl

```diff
@@ -126,21 +121,21 @@ For example, the above conditions imply that:
 - Communicating between iterations using blocking primitives like `Channel`s is incorrect.
 - Write only to locations not shared across iterations (unless a lock or atomic operation is used).
 
-
 Schedule options are:
 - `:static` creates one task per thread and divides the iterations equally among
             them, assigning each task specifically to each thread.
             Specifying `:static` is an error if used from inside another `@threads` loop
             or from a thread other than 1.
-- `:dynamic` will schedule iterations dynamically to available worker threads,
+- `:dynamic` (default) will schedule iterations dynamically to available worker threads,
```

Comment:
Perhaps the default should be listed first

Reply:
makes sense https://github.com/JuliaLang/julia/pull/44163